.github/workflows/rust.yml

```diff
@@ -77,6 +77,6 @@ jobs:
           nix_path: nixpkgs=channel:nixos-22.05
       - uses: cachix/cachix-action@v10
         with:
-          name: mini
-          signingKey: '${{ secrets.CACHIX_SIGNING_KEY }}'
+          name: fedimint
+          signingKey: '${{ secrets.CACHIX_AUTH_TOKEN }}'
```

Comment:
is the attribute still called `signingKey` even though we are using the token auth method?